addons/udes_stock/models/stock_location.py

```diff
@@ -134,6 +134,48 @@ def _domain_height_category(self):
         see u_countable field.""",
     )
 
+    u_heatmap_data_updated = fields.Datetime(
```

Comment:
Could make use of PreciseDatetime here perhaps?

Reply:
How would this affect the timestamp in the incoming location streams data? would it remove the need for the `/1000`?

Reply:
I believe so